src/shared/asset-selector/asset-selector.tsx

```diff
@@ -98,10 +108,12 @@ export const AssetSelector: FC<Props> = ({value, onChange}) => {
                         </div>
                         <div ref={modalSheetRef} className={styles.modalList}>
                             <List
-                                width={listWidth}
+                                // How can we avoid hardcoding this? Need it for margin purposes
```

Comment:
You need an additional div between `List` and `<div ref={modalSheetRef} className={styles.modalList}>` to avoid this